kolibri/plugins/coach/assets/src/views/reports/align-mixin.js

```diff
@@ -0,0 +1,9 @@
+import Vue from 'kolibri.lib.vue';
+
+export default {
+  computed: {
+    align() {
+      return Vue.bidi === 'rtl' ? 'right' : 'left';
+    },
```

Comment:
to make this more general, perhaps two computed props representing the "start" and "end" of a line?

E.g.

```javascript
{
  alignStart() {
    return Vue.bidi === 'rtl' ? 'right' : 'left';
  },
  alignEnd() {
    return Vue.bidi === 'ltr' ? 'right' : 'left';
  },
}
```